save-core/src/commonMain/kotlin/org/cqfn/save/core/utils/ProcessBuilder.kt

```diff
@@ -21,6 +25,103 @@ expect class ProcessBuilder() {
     fun exec(command: List<String>, redirectTo: Path?): ExecutionResult
 }
 
+/**
+ * Class contains common fields for all platforms
+ */
+class ProcessBuilderInternal {
+    /**
+     * Typealias
+     */
+    val fs = FileSystem.SYSTEM
+
+    /**
+     * Temporary directory for stderr and stdout (popen can't separate streams, so we do it ourselves)
+     */
+    val tmpDir = (FileSystem.SYSTEM_TEMPORARY_DIRECTORY / this::class.simpleName!!).also {
```

Comment:
We might run multiple `ProcessBuilder`s concurrently in the future

Reply:
Fixed